cmd/snapd/main_test.go

```diff
@@ -61,29 +61,29 @@ func (s *snapdSuite) SetUpTest(c *C) {
 	s.AddCleanup(restore)
 }
 
-func (s *snapdSuite) TestSanityFailGoesIntoDegradedMode(c *C) {
+func (s *snapdSuite) TestValidityFailGoesIntoDegradedMode(c *C) {
 	logbuf, restore := logger.MockLogger()
 	defer restore()
 	restore = apparmor.MockIsHomeUsingNFS(func() (bool, error) { return false, nil })
 	defer restore()
 	restore = seccomp.MockSnapSeccompVersionInfo("abcdef 1.2.3 1234abcd -")
 	defer restore()
 
-	sanityErr := fmt.Errorf("foo failed")
-	sanityCalled := make(chan bool)
-	sanityRan := 0
-	restore = snapd.MockSanityCheck(func() error {
-		sanityRan++
+	validityErr := fmt.Errorf("foo failed")
+	validityCalled := make(chan bool)
+	validityRan := 0
```

Comment:
I know this is just 's/sanity/validity' but I'd replace "validityCalled" with "checkCalled" and "validityRan" with "checksRan" (or checksCounter)